pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -350,6 +363,25 @@ public Runner build() {
                   vertx, dataDir, jsonRpcConfiguration, metricsSystem, jsonRpcMethods));
     }
 
+    BlockchainQuery queries =
```

Comment:
Wrap all the init in a `if (graphQLConfiguration.isEnabled())` block, like the other services.

Reply:
@shemnon 
thanks for feedback
pls donot pay too much attention on details.  I just want to make sure if the design structure is fit your style?
After all, this WIP need much refactor.  
Will let you know if all details get ready

Reply:
Yes, in general it matches the architecture, which is why most of the comments are stylistic in nature.